sympy/geometry/entity.py

```diff
@@ -97,17 +97,23 @@ def __cmp__(self, other):
         return (i1 > i2) - (i1 < i2)
 
     def __contains__(self, other):
-        """Subclasses should implement this method for anything more complex than equality."""
+        """Subclasses should implement this method for anything more complex than equality.
```

Comment:
I don't think that we need to add the spaces here, especially if it is docs for private or special methods that are not visible.